### ISSUE TITLE
fix: flutter android splash screen is deprecated 

### ISSUE THREAD

**Issue by hawkbee1** [VeryGoodOpenSource/very_good_cli#253]
That's not really a bug.
I created a new project with very good cli. With android emulator the debug console displays : 

**A splash screen was provided to Flutter, but this is deprecated. See flutter.dev/go/android-splash-migration for migration steps.**

**Comment by MindMayhem** [comment]
Remove the below lines from the android manifest file. It's no longer used

```
<meta-data
       android:name="io.flutter.embedding.android.NormalTheme"
       android:resource="@style/NormalTheme"/>
           
 <meta-data
       android:name="io.flutter.embedding.android.SplashScreenDrawable"
       android:resource="@drawable/launch_background"/>

**Comment by marcossevilla** [comment]
hey @hawkbee1 👋 

are you generating a default Flutter app or is it another template like `flutter_plugin`?

**Comment by hawkbee1** [comment]
default app : 
`very_good create ssi_crypto_wallet --desc "SSI and crypto wallet" --org "co.talao.crypto"`
Just checked the version : 0.4.12

**Comment by felangel** [comment]
This has been resolved in [very_good_cli v0.4.14](https://pub.dev/packages/very_good_cli/versions/0.4.14) 🎉
